fabric/build.gradle

```diff
@@ -0,0 +1,80 @@
+apply from: rootProject.file('buildscript/standalone.gradle')
+apply plugin: 'fabric-loom'
+
+configurations.all {
+    resolutionStrategy {
+        force "org.slf4j:slf4j-api:1.7.36" // Introduced by Minecraft itself
+    }
+}
+
+java {
+    disableAutoTargetJvm() // Requires Java 21, we target 8
+}
```

Comment:
The Fabric module can target newer Java as it can only run on MC 1.24.x which only support newer Java versions anyway, something like

```
java {
    sourceCompatibility = JavaVersion.VERSION_21
    targetCompatibility = JavaVersion.VERSION_21
}
```